### PR TITLE
docs(selfhosted): restructure

### PR DESCRIPTION
**Pull request by ghuntley**
**TODO:**
- [ ] Resolve broken links that break the build.

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/773"></a>

### CODE REVIEW

File: src/routes/docs/self-hosted/0.9.0/admin/admin.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-section: self-hosted/latest/self-hosted
```
Comment (geropl) [inline src/routes/docs/self-hosted/0.9.0/admin/admin.md:2]: @ghuntley I see that you adjusted the `0.9.0` sections as well. I don't understand the mechanism here; just wanted to ask and make sure that this is intended?

File: src/routes/docs/self-hosted/latest/configuration/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+section: self-hosted/latest
+title: Configuring Gitpod Self-Hosted
+---
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Database
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/configuration/index.md:10]: Copy'n'paste artifact?

File: src/routes/docs/self-hosted/latest/configuration/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- [Configure the database used by your Gitpod Self-Hosted installation](./database)
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/configuration/index.md:12]: `used by your Gitpod Self-Hosted installation` feels like a lot of noise. :confused:

File: src/routes/docs/self-hosted/latest/configuration/nodes.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
-section: self-hosted/latest/self-hosted
-title: Kubernetes Nodes
+section: self-hosted/latest
+title: Configure the Kubernetes nodes in your Gitpod Self-Hosted installation
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/configuration/nodes.md:3]: again, feels like noise to me.

File: src/routes/docs/self-hosted/latest/index.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Gitpod Self-Hosted
+
+Gitpod, just as you know it from [gitpod.io](https://gitpod.io), can be deployed and operated on your own infrastructure. It supports different cloud providers, self-managed Kubernetes clusters, corporate firewalls, and even off-grid / air-gapped networks.
+
+- [Installing Gitpod Self-Hosted](./latest/installation)
+- [Configuring Gitpod Self-Hosted](./latest//configuration)
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/index.md:15]: `//` -> `/`

File: src/routes/docs/self-hosted/latest/releases/index.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+# Gitpod Self-Hosted releases
+
+## Currently Supported
+
+- [latest](../)
+
+## Deprecated
+
+- [v0.9.0](../0.9.0/self-hosted)
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/releases/index.md:18]: :tada:

File: src/routes/docs/self-hosted/latest/updating/index.md [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
 
 <script context="module">
   export const prerender = true;
 </script>
 
-# Gitpod Self-Hosted Upgrade Notes
+# Updating Gitpod Self-Hosted
 
-## Upgrading Gitpod from v0.10.0 to v0.10.0
+## Upgrading Gitpod from v0.9.0 to v0.10.0
```
Comment (geropl) [inline src/routes/docs/self-hosted/latest/updating/index.md:12]: :+1:

**Comment by geropl** [comment]
Not related to this PR, but a left-over from before: https://github.com/gitpod-io/website/blob/5a7474710fc6555b1b38773c7e26217fdcac8563/src/routes/docs/self-hosted/latest/installation/on-kubernetes.md#L14-L16

File: src/routes/docs/self-hosted/latest/configuration/nodes.md (hunk 4 above)
Comment (ghuntley) [reply src/routes/docs/self-hosted/latest/configuration/nodes.md:3]: Agreed but clearer choice of words isn't coming to me at the moment. Let's get the PR in and iterate from there.

**Pull request merged**